frontend/utility/MissingFilesModel.cpp

```diff
@@ -162,11 +162,17 @@ Qt::ItemFlags MissingFilesModel::flags(const QModelIndex &index) const
 
 void MissingFilesModel::fileCheckLoop(QList<MissingFileEntry> files, QString path, bool skipPrompt)
 {
-	loop = false;
+	fileCheckLoop(files, path, skipPrompt, 0);
+}
+
+void MissingFilesModel::fileCheckLoop(QList<MissingFileEntry> files, QString path, bool skipPrompt, int depth)
+{
+	fileLoopCounter += 1;
 	QUrl url = QUrl().fromLocalFile(path);
```

Comment:
I guess the intention here was to use `QUrl::fromLocalFile` which is a `static` factory function to create a `QUrl` instance with a given `QString`, but instead it creates an empty `QUrl` object first, calls the _static_ method available on that instance, only to discard it when the static method returns the actual instance.

Reply:
Javascript moment